.travis.yml

```diff
@@ -0,0 +1,39 @@
+sudo: false
+
+language: php
+
+cache:
+  directories:
+    - "$HOME/.composer/cache/files"
+
+matrix:
+  include:
+    - php: 5.5
+      env:
+        - COMPOSER_FLAGS="--prefer-dist --no-interaction"
```

Comment:
this is not set to lowest as we want to test the actual sulu version and not 1.6.0